service/src/main/resources/db/changesets/20230410_schema_reset.yaml

```diff
@@ -432,7 +433,7 @@ databaseChangeLog:
                   remarks: Deleting a cohort revision will cascade to delete the criteria contained in it
               - column:
                   name: concept_set_id
-                  type: ${id.type}
+                  type: text
                   constraints:
                     references: concept_set(id)
                     foreignKeyName: fk_crit_cs
```

Comment:
Foreign key name: **fk_crit_cs** in `20230410_schema_reset.yaml#439`  clashes with same name in `20230530_avoid_postgres_specific_features.yaml#74` Please consider renaming one of these to a different name maybe `fk_crit_cs_2` in the `avoid_postgres` file